### PR TITLE
[script.module.autocompletion@matrix] 2.0.5

### PR DESCRIPTION
**Pull request by finkleandeinhorn**
### Add-on details:

- **General**
  - Add-on name: AutoCompletion library
  - Add-on ID: script.module.autocompletion
  - Version number: 2.0.5
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/finkleandeinhorn/script.module.autocompletion
  
Module providing some AutoCompletion functions

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: script.module.autocompletion/.gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Byte-compiled / optimized / DLL files
```
Comment (basrieter) [inline script.module.autocompletion/.gitignore:1]: Don't include

File: script.module.autocompletion/.github/workflows/addon-submitter.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
+name: Kodi Addon-Submitter
```
Comment (basrieter) [inline script.module.autocompletion/.github/workflows/addon-submitter.yml:1]: don't include

**Comment by basrieter** [comment]
@sualfred are you OK with these cahnges?

**Comment by enen92** [comment]
All good, unfortunately all add-ons need to have a maintainer otherwise they are dead. Even if you don't want to be a maintainer please change the upstream repo location so other know where to submit patches before starting to be the new maintainer :)

**Comment by finkleandeinhorn** [comment]
@enen92 Fair enough, I've added my repo as the `<source>`.

**Comment by finkleandeinhorn** [comment]
This needs to be merged in conjunction with https://github.com/xbmc/repo-plugins/pull/4047.

**Review by enen92 — APPROVED** [review]

**Pull request merged**